numpy/linalg/linalg.py

```diff
@@ -1810,7 +1810,7 @@ def det(a):
 
 # Linear Least Squares
 
-def lstsq(a, b, rcond=-1):
+def lstsq(a, b, rcond="warn"):
```

Comment:
At least, that's consistent with how I implemented this kind of thing for the axis argument to MaskedArray.argmin